x-pack/plugins/maps/server/index.ts

```diff
@@ -4,7 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { PluginInitializerContext } from 'src/core/server';
+import { PluginConfigDescriptor } from 'kibana/server';
 import { MapsPlugin } from './plugin';
+import { configSchema, MapsXPackConfig } from '../config';
+
+export const config: PluginConfigDescriptor<MapsXPackConfig> = {
+  exposeToBrowser: {
```

Comment:
Thanks for chatting and clarifying what this is doing. Maybe since this is a little unclear, how about adding the following comments.
```
// exposeToBrowser specifies kibana.yml settings to expose to the browser
// true signals configuration is exposed to browser and is not a default value.
```